src/plugins/displayLayout/components/TelemetryView.vue

```diff
@@ -284,14 +298,31 @@ export default {
         updateTelemetryFormat(format) {
             this.$emit('formatChanged', this.item, format);
         },
-        getContextMenuActions() {
-            let actionsObject = this.openmct.actions.get(this.currentObjectPath, {}, {viewHistoricalData: true});
-            let applicableActionKeys = Object.keys(actionsObject).filter(key => actionsObject[key].key === 'viewHistoricalData');
+        async getContextMenuActions() {
+            const defaultNotebook = getDefaultNotebook();
+
+            const domainObject = defaultNotebook && await this.openmct.objects.get(defaultNotebook.notebookMeta.identifier);
+
+            const actionsObject = this.openmct.actions.get(this.currentObjectPath, this.getViewContext(), { viewHistoricalData: true }).applicableActions;
+            let applicableActionKeys = Object.keys(actionsObject)
+                .filter(key => {
+                    const isViewHistoricalData = actionsObject[key].key === 'viewHistoricalData';
+                    const isCopyToClipboard = actionsObject[key].key === 'copyToClipboard';
+                    const isCopyToNotebook = defaultNotebook && actionsObject[key].key === 'copyToNotebook';
```

Comment:
lets move this to the constant as it was, so in the future, if we needed to add more actions to this menu, we can just update the array at the top